packages/office-ui-fabric-react/src/components/Modal/Modal.base.tsx

```diff
@@ -307,6 +313,51 @@ export class ModalBase extends React.Component<IModalProps, IDialogState> implem
     }
   }
 
+  private _registerInitialModalPosition = (): void => {
+    if (this.props.dragOptions?.keepInBounds && !this._minClampedPosition && !this._maxClampedPosition) {
+      const dialogMain = document.getElementsByClassName('ms-Dialog-main');
```

Comment:
I think if we can it would be better to use a ref here.

Reply:
There could be, although perhaps ill-advised, multiple dialogs in the window at the same time

Reply:
I have replaced that now with data-id attr, since we cannot use the ref here.